src/module/module.py

```diff
@@ -97,9 +101,10 @@ def module_main(received_data: any) -> str:
 
 
 def processData(parsed_data):
+    global LABELS
```

Comment:
why do you need it? mqtt-egress doesn't have this line

Reply:
technically you don't, but it is just an explicit way of saying which LABELS you want to use. It is just a new coding habit that I want to gain 😄

Reply:
where did you get this from??